app/src/main/res/layout/item_susi_link_preview.xml

```diff
@@ -26,15 +25,20 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="right"
-                android:layout_weight="1">
+                android:layout_weight="1"
+                android:paddingTop="15dp"
+                android:paddingRight="15dp"
```

Comment:
Use dimens.xml to specify size instead of hard coding it in layouts. It is present in many places in this PR. Please correct them as well.